.gitattributes

```diff
@@ -1,2 +1,15 @@
+###############################################################################
```

Comment:
This file is the only additions (the +13 lines), everything else is just line ending changes